integration/tests/cook/util.py

```diff
@@ -483,4 +483,5 @@ def progress_line(cook_url, percent, message):
         raise Exception('{} not present in {} regex string'.format('(\d+)', regex_string))
     if not '(.*)' in regex_string:
         raise Exception('{} not present in {} regex string'.format('(.*)', regex_string))
-    return regex_string.replace('(\d+)', str(percent)).replace('(.*)', str(message)).replace('(?: )?', ' ')
+    simplified_regex_string = regex_string.replace('(?: )?', ' ').replace('\\', '')
+    return simplified_regex_string.replace('(\d+)', str(percent)).replace('(.*)', str(message))
```

Comment:
I think you need to replace the backslash *last*, otherwise it's going to clobber the backslashes in your other patterns (e.g., `(\d+)` → `(d+)`, which will probably cause the test to fail).

Nitpick: I'd prefer this all as one expression. You can force Python to treat multiple lines as a single expression if you wrap it all in parentheses:
```python
return (regex_string
        .replace('(\d+)', str(percent))
        .replace('(.*)', str(message))
        .replace('(?: )?', ' ')
        .replace('\\', ''))